globalize-accessors.gemspec

```
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3"
   s.rubyforge_project         = "globalize-accessors"
 
+  s.add_dependency 'activesupport', '>= 4.2', '< 7.2'
```

```suggestion
  s.add_dependency 'activesupport', '>= 4.2', '< 8.2'
```

---

Maybe it should just be
```suggestion
  s.add_dependency 'activesupport', '>= 4.2'
```

To avoid having to update it again.

---

I have removed the upper limit.